src/components/photo_gallery/photo_gallery.module.css

```diff
@@ -0,0 +1,90 @@
+.wrapper {
+  width: 100%;
+  max-width: 1140px;
+  margin: 0 auto;
+}
+
+.list {
+  display: grid;
+  grid-gap: 10px;
```

Comment:
could grid-gap value be moved into scc var and used here and below to calculate values?